projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesExpSimpleMoneynessProvider.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (C) 2014 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.analytics.financial.provider.description.interestrate;
+
+import java.util.List;
+import java.util.Set;
+
+import org.apache.commons.lang.ObjectUtils;
+
+import com.opengamma.analytics.financial.instrument.index.IborIndex;
+import com.opengamma.analytics.financial.provider.sensitivity.multicurve.ForwardSensitivity;
+import com.opengamma.analytics.math.surface.Surface;
+import com.opengamma.util.ArgumentChecker;
+import com.opengamma.util.tuple.DoublesPair;
+
+/**
+ * Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent.
+ * The "delay" is the time between expiration of the option and last trading date of the underlying futures.
```

Comment:
Change Javadoc to a more specific explanation:
" Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to is time to expiration/simple moneyness dependent. The simple moneyness is computed as the difference between the strikePrice and the futuresPrice."